upgrading/upgrade-operator.adoc

```diff
@@ -7,7 +7,11 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-Upgrades through the {rh-rhacs-first} Operator are performed automatically or manually, depending on the *Update approval* option you chose at installation.
+Upgrades through the {rh-rhacs-first} Operator are performed automatically or manually, depending on the *Update approval* option you chose at installation. Upgrading from version 3.74 to 4.x requires some additional steps that are described on this page. Upgrading within 4.y versions requires no additional steps.
+
+[discrete]
+[id="upgrading-from-version-3x-to-4x_{context}"]
+== Upgrading from version 3.74 to 4.x
```

Comment:
I'm worried, this title `Upgrading from version 3.74 to 4.x` may not be entirely accurate, since `Remove Central-attached PV` section applies to upgrading to 4.1 from _any_ previous version, including 4.0. That said, it looks both `Upgrading from version 3.74 to 4.x` and `Remove Central-attached PV` sections are at the same level of hierachy (both are `h2` tags in rendered HTML). Which is maybe fine if we mean that `Remove Central-attached PV` applies to upgrade from _any_ to 4.1. However, `Changing subscription channel` **logically belongs** to `Upgrading from version 3.74 to 4.x` procedure but is also `h2` tag, i.e. when rendered, it appears separate. `Modifying Central custom resource` is a step in 3.x -> 4.x upgrade, `Modifying Central custom resource for external database` is an optional step in 3.x -> 4.x upgrade also.

So it looks like a mess, and I wonder how do users comprehend that. (Shared [same thoughts](https://issues.redhat.com/browse/ROX-18594?focusedId=22799122&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-22799122) in the ticket rox-18594)

Reply:
So earlier you said that no additional instructions are needed for upgrades within 4.x, so I assumed this page was only for 3.74 - 4.x, as noted in the third sentence of the page. I've tried to play around with the levels and see if that helps alleviate the confusion. I'll push those changes soon.

Reply:
_I'm worried, this title Upgrading from version 3.74 to 4.x may not be entirely accurate, since Remove Central-attached PV section applies to upgrading to 4.1 from any previous version, including 4.0._

But it does apply to 3.74 -> 4.1, so it's ok for it to be here, right? Basically, if we want to have separate pages for 3.74 ->4.x, and within 4.x, we will need to split out the files further and create new assembly files, and I was trying to avoid doing that if we really don't need to provide special instructions for upgrades within 4.x. But if that needs a page of its own, I can create one and include the "Remove central-attached PV" sections.

Reply:
Looks like your https://github.com/openshift/openshift-docs/pull/63530 supersedes this change, I suggest continuing the discussion there.